types/common/address.go

```diff
@@ -129,6 +129,11 @@ func HexToHeimdallAddress(b string) HeimdallAddress {
 	return HeimdallAddress(common.HexToAddress(b))
 }
 
+// HexToHeimdallAddress returns Address with value b.
+func HexToAccAddress(b string) sdk.AccAddress {
+	return sdk.AccAddress(b)
+}
+
 // AccAddressToHeimdallAddress returns Address with value b.
 func AccAddressToHeimdallAddress(b sdk.AccAddress) HeimdallAddress {
```

Comment:
Let's remove `HeimdallAddress` slowly in future PRs.

Reply:
Removed the use of `HexToAccAddress` in the latest commit